databricks_sync/sdk/report/model.py

```diff
@@ -0,0 +1,277 @@
+import base64
+import datetime
+import os
+import uuid
+from pathlib import Path
+from typing import List, Optional
+
+import pandas as pd
+from sqlalchemy import create_engine, Column, String, DateTime, UniqueConstraint, and_, update, text
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from styleframe import StyleFrame, utils, Styler
+from tabulate import tabulate
+
+from databricks_sync import log
+
+driver = os.environ.get("DATABRICKS_SYNC_REPORT_DB_DRIVER", "sqlite")
+path = os.environ.get("DATABRICKS_SYNC_REPORT_DB_URL",
+                      None)
+echo = os.environ.get("DATABRICKS_SYNC_REPORT_DB_TRACE",
+                      "false")
+if path is None:
+    default_db_path = Path.home() / '.databricks_sync/'
+    default_db_path.mkdir(parents=True, exist_ok=True)
+    default_db_name = "report.db"
+    path = str(default_db_path/default_db_name)
+
+db_connection_string = f'{driver}:///{path}'
```

Comment:
Do we need to add validation in here or in another place?
Or will it just error out?